h2o-core/src/main/java/water/fvec/task/UniqOldTask.java

```diff
@@ -0,0 +1,53 @@
+package water.fvec.task;
+
+import water.MRTask;
+import water.fvec.Chunk;
+import water.fvec.Vec;
+import water.rapids.ast.prims.mungers.AstGroup;
+import water.util.IcedHashMap;
+
+@Deprecated // in favor of UniqTask - kept for experimenting only
+public class UniqOldTask extends MRTask<UniqOldTask> {
```

Comment:
I'd kill it right away. No need to experiment with it. If this is to demonstrate a technical concept, we're better rename & move it. No big deal though.

Reply:
Yeah, I know what you mean... I kept it for experiments for our support team. Given the current uncertainty whether there is a possible issue with unique implementation - I want to give power-users a chance to experiment with old vs new on a nightly build and be able to easily prove there is an improvement. I will remove once these experiments are done.